debug/src/debug.js

```diff
@@ -6,6 +6,23 @@ export function initDebug() {
 	/* eslint-disable no-console */
 	let oldBeforeDiff = options.diff;
 
+	options.checkParent = parentNode => {
+		if (!parentNode) {
+			throw new Error('Undefined parent passed to render(), this is the second argument.\nCheck if the element is available in the DOM/has the correct id.');
+		}
+		let isValid;
+		switch (parentNode.nodeType) {
+			case 1:
```

Comment:
Maybe we can add constants here. I never can remember which number stands for which `nodeType` :slightly_smiling_face: